src/Test/AbstractFixerTestCase.php

```diff
@@ -34,7 +35,7 @@
     protected $linter;
 
     /**
-     * @var null|FixerInterface
+     * @var null|FixerInterface|ConfigurableFixerInterface
```

Comment:
if some class is implementing `ConfigurableFixerInterface` but **not** `FixerInterface` it will fail here.

Why is this change ?